message-index/messages/GHC-02256/differentmodule/after/Module2.hs

```diff
@@ -0,0 +1,9 @@
+{-# LANGUAGE DuplicateRecordFields #-}
+module Module2 where
+
+import Module1 ( R1(x) )
```

Comment:
You don't necessarily need this import to be qualified, do you? Although I suppose if `Module1` exported two `x` fields then you would...

It could be worth demonstrating the trick
```hs
import qualified Module1 ( R1(x) ) as R1
```
and then using `R1.x` to refer to the field?